pkg/infra/tower.go

```diff
@@ -100,6 +100,7 @@ func (t *Tower) endOfDay() {
 	t.dayInfo.CurrDay += 1
 	for _, agent := range t.Agents {
 		agent := agent.BaseAgent()
+		t.stateLog.LogUtility(t.dayInfo, agent.agentType, agent.Utility(), agent.IsAlive())
```

Comment:
I think `IsAlive` is updated after hp decay I called on the line after

Reply:
I see I see, will change